frontend/ui/widget/toggleswitch.lua

```diff
@@ -211,8 +211,14 @@ end
 
 function ToggleSwitch:onHoldSelect(arg, gev)
     local position = self:calculatePosition(gev)
-    self.config:onMakeDefault(self.name, self.name_text,
-                    self.values or self.args, self.toggle, position)
+    if self.name == "font_fine_tune" then
+        --- @note Ugly hack for the only widget that uses a dual toggle for fine-tuning (others prefer a buttonprogress)
```

Comment:
I also still have to make that fine-tuning thing work for EPUB/FB2 in MuPDF. :-P